space-template.yaml

```diff
@@ -0,0 +1,166 @@
+        en_US: "A work item can have different states (e.g. new, open, or closed)."
+        de_DE: "Ein Fehler kann mehrere Zustände besitzen."
+    type: *list
+    element_type: *translatable_string
+    default_value: *state_new
+    values: *common_work_item_states
+    required: false
+
+# If no other fields are required for your work item you can use this handy default
+common_work_item_fields: &common_work_item_fields
+  system_title: *field_title
+  system_description: *field_description
+  system_created_at: *field_created_at
+  system_updated_at: *field_updated_at
+  system_state: *field_state
+
+# This map defines all the work item types that exist in a space. We re-use some
+# of the above states and lists in order share code between work item types.
+work_item_types:
+  &wit_bug bug:
+    title:
+      en_US: "Bug"
+      de_DE: "Fehler"
+    description:
+      en_US: "A software bug is an error, flaw, failure or fault in a computer program"
+      de_DE: "Ein Fehler bezeichnet im Allgemeinen ein Fehlverhalten von Computerprogrammen"
+    fields: *common_work_item_fields
+  &wit_feature feature:
+    title:
+      en_US: "Feature"
+      de_DE: "Fähigkeit"
+    description:
+      en_US: "TBD"
+      de_DE: "TBD"
+    fields: *common_work_item_fields
+  &wit_epic epic:
+    title:
+      en_US: "Epic"
+      de_DE: "Epik"
+    description:
+      en_US: "TBD"
+      de_DE: "TBD"
+    fields: *common_work_item_fields
+  &wit_user_story user_story:
+    title:
+      en_US: "User story"
+      de_DE: "Anwendererzählung"
+    description:
+      en_US: "A way to describe a scenario such that it has clear acceptance 
+      criteria and that unambiguously describes a user interaction , or the
+      interaction between two parts of a system. Team members tak a user story
+      and decompose it into tasks that satisfy the user story. Usually some
+      point value is assigned to a story that serves a way to measure the
+      rate of completing user stories."
+      de_DE: ""
+    fields: *common_work_item_fields
+  &wit_task task:
+    title:
+      en_US: "Task"
+      de_DE: "Aufgabe"
+    description:
+      en_US: "TBD"
+      de_DE: "TBD"
+    fields: *common_work_item_fields
+  &wit_scenario scenario:
+    title:
+      en_US: "Scenario"
+      de_DE: "Scenario"
+    description:
+      en_US: "A specific defined interaction/sequence of interactions with a
+             system to achieve a given goal"
+      de_DE: "Eine speziell definierte Interaktion/Sequenz von Interaktionen
+             mit einem System, um ein Ziel zu erreichen"
+    fields: *common_work_item_fields
+  &wit_experience experience:
+    title:
+      en_US: "Experience"
+      de_DE: "Erfahrung"
+    description:
+      en_US: "TBD"
+      de_DE: "TBD"
+    fields: *common_work_item_fields
+
+work_item_categories:
+    requirements: [*wit_task, *wit_bug]
+    board: [*wit_user_story, *wit_task, *wit_bug]
+    root_level: [*wit_scenario, *wit_experience]
```

Comment:
having a "root_level" seem incorrect to me. 

That should be contextual to what view you are on.

i.e. a board view should list the category types it would want to use for its root levels. i.e. I would not expect scenarios/experiences to show up in the daily board scrum views - but I would expect them on a product focused board.